packages/rpc/src/methods/keys.test.ts

```diff
@@ -0,0 +1,22 @@
+import fetch from 'isomorphic-unfetch';
+import { getAccessKeys } from './keys';
+
+jest.mock('isomorphic-unfetch');
+
+describe('keys', () => {
```

Comment:
Could we also add tests that the errors are thrown when the response fields don't exist? That way we would know if someone accidentally deletes one of those "throws".

Reply:
Yup, done